paasta_tools/chronos_tools.py

```diff
@@ -191,6 +192,12 @@ def read_chronos_jobs_for_service(service, cluster, soa_dir=DEFAULT_SOA_DIR):
 
 
 def load_chronos_job_config(service, instance, cluster, load_deployments=True, soa_dir=DEFAULT_SOA_DIR):
+    log.info("Reading general configuration file: service.yaml")
```

Comment:
Is this really necessary?
You could put this at the debug level, but I say don't bother logging this